src/stores/studyStore.ts

```diff
@@ -164,25 +164,35 @@ export const useStudyStore = defineStore('study', () => {
       )
       .then(async (rs) => {
         if (rs && rs?.data?.token) {
-          await importExportApi.exportStudyData(
-            studyId,
-            rs.data.token,
-            studyGroupId,
-            participantId,
-            observationId,
-            from,
-            to
-          ).then((response) => {
-            window.open(response.headers.location);
-            const filename: string = 'study_data_' + studyId + '.json';
-            downloadJSON(filename, response.data);
+          await importExportApi
+            .exportStudyData(
+              studyId,
+              rs.data.token,
+              studyGroupId,
+              participantId,
+              observationId,
+              from,
+              to,
+              { responseType: 'blob' },
+            )
+            .then((response) => {
+              const blob = response.data as unknown as Blob;
 
-          }).catch((e: AxiosError) => {
-            handleIndividualError(
-              e,
-              'cannot export data despite of existing download token',
-            );
-          } );
+              const url = URL.createObjectURL(blob);
+              const link = document.createElement('a');
+              link.href = url;
+              link.download = `study_data_${studyId}.json`;
```

Comment:
Wäre es nicht besser, wenn das Webservice schon einen passenden `Content-Disposition` Header setzten würde. z.B.

```
Content-Disposition: attachment; filename="study_data_study-{id}.json"
```
Würde in den Namen auch gleich noch andere Filter wie StudyGroup oder Participant sowie die Time-Range mit reincodieren (am Backend). Dieser Header sollte dann den Browser dazu veranlassen das Ergebnis direkt in einem File zu speichern